### PR TITLE
LPS-174037 Remove duplicated categories

### PR DESCRIPTION
**Pull request by victorg1991**

### CODE REVIEW

**Comment by liferay-echo** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

**Comment by ruben-pulido** [comment]
I think it would make sense to deduplicate the ids when first setting them in ServiceContext.

https://github.com/liferay/liferay-portal/blob/324a611760a954902a602c3ac0737f1a9e9a42c4/portal-kernel/src/com/liferay/portal/kernel/service/ServiceContextFactory.java#L333

File: portal-kernel/src/com/liferay/portal/kernel/service/ServiceContextFactory.java [hunk 1]
```diff
@@ -317,7 +319,7 @@ private static ServiceContext _getInstance(
 		Map<String, String[]> parameterMap =
 			httpServletRequest.getParameterMap();
 
-		List<Long> assetCategoryIdsList = new ArrayList<>();
+		Set<Long> assetCategoryIdsList = new HashSet<>();
```
Comment (ruben-pulido) [inline portal-kernel/src/com/liferay/portal/kernel/service/ServiceContextFactory.java:322]: Shouldn't we name the variable assetCategoryIdsSet now?

**Comment by victorg1991** [comment]
ci:reevaluate:1377401_8514

**Comment by victorg1991** [comment]
ci:forward

**Comment by Tim-Cao** [comment]
See https://github.com/brianchandotcom/liferay-portal/pull/129515

**Comment by Tim-Cao** [comment]
ci:close

**Pull request closed without merging**